sdk/keyvault/azure-security-keyvault-certificates/src/samples/README.md

```diff
@@ -0,0 +1,105 @@
+
+## Azure Key Vault Certificate Samples client library for Java
+This document explains samples and how to use them.
+
+## Key concepts
```

Comment:
Key concepts and Getting started sections in samples should just have a link to the root README.